pkgs/tools/admin/okta-aws-cli/default.nix

```diff
@@ -0,0 +1,26 @@
+{ lib, buildGoModule, fetchFromGitHub }:
+
+buildGoModule rec {
```

Comment:
The package seems to build many subpacakges but only installs one

```
Building subPackage ./cmd/okta-aws-cli
Building subPackage ./cmd/root
Building subPackage ./internal/agent
Building subPackage ./internal/ansi
Building subPackage ./internal/aws
Building subPackage ./internal/backoff
Building subPackage ./internal/config
Building subPackage ./internal/output
Building subPackage ./internal/sessiontoken
```

```
ls result/bin
okta-aws-cli
```

consider specifying 

```nix
subPackages = [ "cmd/okta-aws-cli" ];
```

to limit what it's building.